src/common/components/Alert/Alert.module.scss

```diff
@@ -0,0 +1,91 @@
+.alert {
+  position: absolute;
+  top: 0;
+  left: -200%;
+  bottom: 0;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  opacity: 0;
+  transition: 0.2s opacity ease-in;
+
+  // Animated gradients from https://www.gradient-animator.com
+  background: linear-gradient(270deg, #ffea00, #ffa600);
+  background-size: 400% 400%;
+
+  -webkit-animation: winner 23s ease infinite;
+  -moz-animation: winner 23s ease infinite;
+  animation: winner 23s ease infinite;
+
+  @-webkit-keyframes winner {
```

Comment:
`create-react-app`, which this app is built from, comes with autoprefixing, so you don't eed to handle prefixes for other browsers.

https://create-react-app.dev/docs/post-processing-css/